lambdas/service/app.py

```diff
@@ -949,24 +949,30 @@ def create_cart():
     }
 
 
-# TODO: implement default cart (may need to change get_all_carts() endpoint)
 @app.route('/resources/carts/{cart_id}', methods=['GET'], cors=True)
 def get_cart(cart_id):
     """
     Get the cart of the given ID belonging to the user
 
-    Returns a 404 error if the cart does not exist or does not belong to the user
+    The default cart can be accessible by using the actual cart UUID or
+    setting the requesting cart ID to "default". If the default cart does
+    not exist, the endpoint will create one automatically.
```

Comment:
In REST, GET should not have persistent side effects, if at all possible. Would it be difficult to NOT create a default cart during GET requests?